pyproject.toml

```diff
@@ -31,15 +31,16 @@ itsdangerous = "^2.1.2"
 starlette-oauth2-api = "^0.2.6"
 httpx = "^0.20.0"
 
-#fasttext = "^0.9.2"  # for window users,  use fasttext-wheel = "^0.9.2"
+fasttext = "^0.9.2"  # for window users,  use fasttext-wheel = "^0.9.2"
```

Comment:
I had forgotten this was generally broken with poetry.  It actually works but yields a bunch of error messages when installing.  =\